src/d3a/models/strategy/area_agents/one_sided_agent.py

```diff
@@ -26,15 +26,20 @@ class OneSidedAgent(InterAreaAgent):
     def __init__(self, *, owner, higher_market, lower_market,
                  min_offer_age=ConstSettings.IAASettings.MIN_OFFER_AGE,
                  engine_type=IAAEngine):
-        super().__init__(engine_type=engine_type, owner=owner, higher_market=higher_market,
-                         lower_market=lower_market, min_offer_age=min_offer_age)
+        super().__init__(engine_type=engine_type, owner=owner,
+                         higher_market=higher_market,
+                         lower_market=lower_market,
+                         min_offer_age=min_offer_age)
         self.name = make_iaa_name(owner)
 
     def usable_offer(self, offer):
         """Prevent IAAEngines from trading their counterpart's offers"""
         return all(offer.id not in engine.forwarded_offers.keys() for engine in self.engines)
 
     def _get_market_from_market_id(self, market_id):
+        if ConstSettings.GeneralSettings.EVENT_DISPATCHING_VIA_REDIS:
```

Comment:
This is not needed here as well (probably leading to an error)

Reply:
Thanks, removing this